xfce-base/exo/files/exo-4.16.0-allow-cross.patch

```diff
@@ -0,0 +1,10 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -151,6 +151,7 @@
+   }
+ ], [
+   AC_DEFINE([HAVE_STRFTIME_EXTENSION], 1, [Define if strftime supports %E and %O modifiers.])
++],,[AC_MSG_WARN([skipping strftime test due to cross-compiling])
```

Comment:
1. Please send this patch upstream and gimme the link.
2. I think you should default to `1`. According to `strftime(3)` both `%E` and `%O` are SUS things, so most systems should really have them. Ideally, exo would test this at runtime and run appropriate code.

Reply:
I opened https://gitlab.xfce.org/xfce/exo/-/issues/52 and changed the patch to default-enable for glibc.